### PR TITLE
added missing meta descriptions to 9 files

### PR DESCRIPTION
**Pull request by Bradben**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0e17482fa3f5cae6822d5661ac468882da496a45/articles/install-guide/index.md) | 90 | 74 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/95cfb0e4-ba8d-4431-b333-8d0cefde2e15_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0e17482fa3f5cae6822d5661ac468882da496a45/articles/install-guide/update.md) | 85 | 62 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/423f653e-e78c-430d-b7cd-d2b03d492041_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/schema/broombridge.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0e17482fa3f5cae6822d5661ac468882da496a45/articles/libraries/chemistry/schema/broombridge.md) | 65 | 26 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/06795296-c72e-4650-82fa-62d6f2349e1e_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/schema/spec_v_0_1.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0e17482fa3f5cae6822d5661ac468882da496a45/articles/libraries/chemistry/schema/spec_v_0_1.md) | 82 | 66 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0d730eec-4918-4917-b0c6-c9a74e69acba_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/schema/spec_v_0_2.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0e17482fa3f5cae6822d5661ac468882da496a45/articles/libraries/chemistry/schema/spec_v_0_2.md) | 85 | 69 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d9956046-c106-4062-aeb9-6330c8974793_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0e17482fa3f5cae6822d5661ac468882da496a45/articles/libraries/index.md) | 62 | 49 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/40565a00-c0e1-45a5-8c4e-d84249c292b7_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/overview/how-to-learn-quantum-computing.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0e17482fa3f5cae6822d5661ac468882da496a45/articles/overview/how-to-learn-quantum-computing.md) | 84 | 75 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/de5077f1-0e11-4e0d-a73b-5244905cc9d4_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/overview/why-learn-quantum-computing.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0e17482fa3f5cae6822d5661ac468882da496a45/articles/overview/why-learn-quantum-computing.md) | 87 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0e124635-4571-4416-a4da-0abc94896038_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/relnotes/0.3-migration.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0e17482fa3f5cae6822d5661ac468882da496a45/articles/relnotes/0.3-migration.md) | 78 | 60 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4fd6b9bb-e591-4be1-bafd-db16b6a99ccc_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/libraries/chemistry/schema/broombridge.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Broombridge - Quantum Chemistry Schema
+description: Overview of the Boombridge Quantum Chemistry Schema, used to model real-world chemistry problems with the Microsoft Quantum Development Kit.
```
Comment (cgranade) [inline articles/libraries/chemistry/schema/broombridge.md]:
```suggestion
description: Overview of the Broombridge quantum chemistry schema, used to model real-world chemistry problems with the Microsoft Quantum Development Kit. 
```

File: articles/libraries/chemistry/schema/spec_v_0_1.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
-title: Broombridge Schema Specification
+title: Broombridge Schema Specification v0.1
+description: Details the specifications for the Broombridge Quantum Chemistry Schema v0.1 for the Microsoft Quantum Chemistry library.
```
Comment (cgranade) [inline articles/libraries/chemistry/schema/spec_v_0_1.md]:
```suggestion
description: Details the specifications for the Broombridge quantum chemistry schema v0.1 for the Microsoft quantum chemistry library. 
```

File: articles/libraries/chemistry/schema/spec_v_0_2.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
-title: Broombridge Schema Specification
+title: Broombridge Schema Specification v0.2
+description: Details the specifications for the Broombridge Quantum Chemistry Schema v0.2 for the Microsoft Quantum Chemistry library.
```
Comment (cgranade) [inline articles/libraries/chemistry/schema/spec_v_0_2.md]:
```suggestion
description: Details the specifications for the Broombridge quantum chemistry schema v0.2 for the Microsoft quantum chemistry library. 
```

File: articles/libraries/index.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
-title: Quantum Development Kit Libraries | Microsoft Docs
+title: Quantum Development Kit Libraries
+description: Overview of the Standard, Chemistry, and Numerics libraries included in the Microsoft Quantum Development Kit.
```
Comment (cgranade) [inline articles/libraries/index.md]:
```suggestion
description: Overview of the standard, chemistry, and numerics libraries included in the Microsoft Quantum Development Kit.
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8058817d396750c39540792e4dae90423e3b657c/articles/install-guide/index.md) | 90 | 74 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/335729b8-e473-41c3-86b0-f830d8c959b2_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8058817d396750c39540792e4dae90423e3b657c/articles/install-guide/update.md) | 85 | 62 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f9e3d517-4d38-4f02-8259-0cfecbcb732f_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/schema/broombridge.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8058817d396750c39540792e4dae90423e3b657c/articles/libraries/chemistry/schema/broombridge.md) | 65 | 26 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/45be830e-3009-4bd2-aa16-9fe7d070d3d2_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/schema/spec_v_0_1.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8058817d396750c39540792e4dae90423e3b657c/articles/libraries/chemistry/schema/spec_v_0_1.md) | 82 | 66 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d130fcf2-01f8-42e8-bb16-29df815f32a6_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/schema/spec_v_0_2.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8058817d396750c39540792e4dae90423e3b657c/articles/libraries/chemistry/schema/spec_v_0_2.md) | 85 | 69 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/553c8090-96cf-4679-9f69-b5032a1bc569_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8058817d396750c39540792e4dae90423e3b657c/articles/libraries/index.md) | 62 | 49 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5f9797f0-4ea6-4fbb-b590-9044b2ee3e9e_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/overview/how-to-learn-quantum-computing.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8058817d396750c39540792e4dae90423e3b657c/articles/overview/how-to-learn-quantum-computing.md) | 84 | 75 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c8e61de0-845a-4b03-aa3b-e2fc46166c2f_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/overview/why-learn-quantum-computing.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8058817d396750c39540792e4dae90423e3b657c/articles/overview/why-learn-quantum-computing.md) | 87 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/255cff2a-b33d-40d9-bf5c-138291cf3db4_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/relnotes/0.3-migration.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8058817d396750c39540792e4dae90423e3b657c/articles/relnotes/0.3-migration.md) | 78 | 60 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7a2ffd98-12ab-4cf9-873c-18ce9f515d18_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0e16cf9e4aede52a93d812d3126ed54219d5996a/articles/install-guide/index.md) | 90 | 74 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2094376f-1f2b-44a2-b09c-3b97ed8fc67d_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0e16cf9e4aede52a93d812d3126ed54219d5996a/articles/install-guide/update.md) | 85 | 62 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d4bf4d4d-529e-4155-a309-3152af28e1cc_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/schema/broombridge.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0e16cf9e4aede52a93d812d3126ed54219d5996a/articles/libraries/chemistry/schema/broombridge.md) | 65 | 26 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/be987b25-1f06-490b-acb9-bf913667da8c_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/schema/spec_v_0_1.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0e16cf9e4aede52a93d812d3126ed54219d5996a/articles/libraries/chemistry/schema/spec_v_0_1.md) | 82 | 66 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/def480f6-4284-4510-b2c0-09b715e8abf2_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/schema/spec_v_0_2.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0e16cf9e4aede52a93d812d3126ed54219d5996a/articles/libraries/chemistry/schema/spec_v_0_2.md) | 85 | 69 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/74b9fb11-6120-418c-957c-8c6d91127635_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0e16cf9e4aede52a93d812d3126ed54219d5996a/articles/libraries/index.md) | 62 | 49 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/21eef6ed-f111-465b-aa88-dbabce23d925_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/overview/how-to-learn-quantum-computing.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0e16cf9e4aede52a93d812d3126ed54219d5996a/articles/overview/how-to-learn-quantum-computing.md) | 84 | 75 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c5735fcc-897f-4407-a186-cfab4f3402a8_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/overview/why-learn-quantum-computing.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0e16cf9e4aede52a93d812d3126ed54219d5996a/articles/overview/why-learn-quantum-computing.md) | 87 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b0ad32b9-5a48-4ec4-9b63-5ca589c6731b_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/relnotes/0.3-migration.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0e16cf9e4aede52a93d812d3126ed54219d5996a/articles/relnotes/0.3-migration.md) | 78 | 60 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0ac5979d-4a00-431b-98d2-a2f7df725da4_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/install-guide/index.md) | 90 | 74 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/285c481e-cb5b-45ae-af8e-ea04df4e0295_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/install-guide/update.md) | 85 | 62 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5dfbacc3-41d5-4991-856c-6771756995b0_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/schema/broombridge.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/libraries/chemistry/schema/broombridge.md) | 65 | 26 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6dcc14d7-6e30-4285-ba49-ce95a369ec9f_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/schema/spec_v_0_1.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/libraries/chemistry/schema/spec_v_0_1.md) | 82 | 66 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/03da9db7-c8fc-44c4-934c-590d2038ea5a_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/schema/spec_v_0_2.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/libraries/chemistry/schema/spec_v_0_2.md) | 85 | 69 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ac7a8575-4cf4-4f53-8627-81e1f1f77ce7_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/libraries/index.md) | 62 | 49 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/101056a6-65b2-4f39-bed1-235ed351ff6b_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/overview/how-to-learn-quantum-computing.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/overview/how-to-learn-quantum-computing.md) | 84 | 75 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d25a7fcd-12f4-4a6f-83e1-126566d70da6_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/overview/why-learn-quantum-computing.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/overview/why-learn-quantum-computing.md) | 87 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/50abe327-2f77-4fe0-b8ae-1a36ecdeb8bc_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/relnotes/0.3-migration.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/relnotes/0.3-migration.md) | 78 | 60 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e098daf8-f143-400b-8507-2a68fa34bb72_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/install-guide/index.md) | 90 | 74 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4b512943-f343-4e16-a28f-77feeaf8b5ab_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/install-guide/update.md) | 85 | 62 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ff5e1057-d9a0-448f-87bc-a32e589beec4_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/schema/broombridge.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/libraries/chemistry/schema/broombridge.md) | 65 | 26 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/174af42b-20a7-41cf-bb93-9e9123006ef3_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/schema/spec_v_0_1.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/libraries/chemistry/schema/spec_v_0_1.md) | 82 | 66 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/88d1c3a4-bcc7-4163-b83f-fb42a50e52f1_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/schema/spec_v_0_2.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/libraries/chemistry/schema/spec_v_0_2.md) | 85 | 69 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e9050d64-c4e9-43fb-b926-db57e6ecb396_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/libraries/index.md) | 62 | 49 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a3f8d358-270e-4322-8b0e-65824b6f43a3_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/overview/how-to-learn-quantum-computing.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/overview/how-to-learn-quantum-computing.md) | 84 | 75 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b5d9bfff-4f05-4b30-92f3-da27ae9c018f_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/overview/why-learn-quantum-computing.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/overview/why-learn-quantum-computing.md) | 87 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4d3b504e-0284-456f-b4c3-3bd01e10bc34_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/relnotes/0.3-migration.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/relnotes/0.3-migration.md) | 78 | 60 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f22e9f84-f0fc-4517-88ee-c236eb2e4234_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/install-guide/index.md) | 90 | 74 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b7d059bf-20dc-4dc9-97bc-66482ef85999_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/install-guide/update.md) | 85 | 62 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d62740ae-9e49-44b1-9c9b-23852237d28b_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/schema/broombridge.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/libraries/chemistry/schema/broombridge.md) | 65 | 26 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d9e61454-df66-4cf9-aff8-2ac2f4ef993d_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/schema/spec_v_0_1.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/libraries/chemistry/schema/spec_v_0_1.md) | 82 | 66 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0e469633-09fe-4ab1-bbfd-9745e23b5354_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/schema/spec_v_0_2.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/libraries/chemistry/schema/spec_v_0_2.md) | 85 | 69 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b9d02dff-1b67-4b34-b512-a68dc9af5b42_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/libraries/index.md) | 62 | 49 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/70385987-4ceb-42fc-bfc2-d13990173ca7_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/overview/how-to-learn-quantum-computing.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/overview/how-to-learn-quantum-computing.md) | 84 | 75 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/61aef9ae-bb14-4ef4-bd42-221d29d21e21_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/overview/why-learn-quantum-computing.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/overview/why-learn-quantum-computing.md) | 87 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/96464359-7966-45c7-a132-3a1d83f2863d_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/relnotes/0.3-migration.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8881eed50dd47289a70473b43106525c78821c16/articles/relnotes/0.3-migration.md) | 78 | 60 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/73ceb8cd-863f-4232-9c5b-33d1bfa2281f_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Pull request merged**